packages/cross_file/test/image_data.dart

```diff
@@ -0,0 +1,132 @@
+/// A 50x50 blue square png.
+const List<int> kBlueSquarePng = <int>[
```

Comment:
This is copied from https://github.com/flutter/flutter/blob/125545da322048565911bf81118cce60f05036ce/packages/flutter/test/image_data.dart#L6